librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -155,7 +157,48 @@ void QG_CommandWidget::tabPressed() {
             leCommand->setText(reducedChoice.first());
         }
         else if (reducedChoice.count()>0) {
+            QString longestString = "";
+            QString shortestString = "";
```

Comment:
the initialization is not necessary, because default ctor of QString generates an empty string